app/controllers/resources_controller.rb

```diff
@@ -4,7 +4,7 @@ class ResourcesController < ApplicationController
   def index
     category_id = params.require :category_id
     relation = resources.joins(:categories).joins(:address)
-                        .where('categories.id' => category_id).where('status' => :approved)
+                        .where('categories.id' => category_id).where(status: Resource.statuses[:approved])
```

Comment:
I don't understand why this wasn't working before. The [ActiveRecord docs](http://api.rubyonrails.org/classes/ActiveRecord/Enum.html) explicitly say that you can do this:

```rb
Conversation.where(status: [:active, :archived])
Conversation.where.not(status: :active)
```

Reply:
Those were working for me as well, to be safe I went with the most explicit version I saw that worked.

Interestingly though, 'where('status' => :approved)' works in the dev environment but not on staging.

¯\_(ツ)_/¯

My only suspicion currently is the difference in pre-compiling ruby before it goes to staging, otherwise versions are the same.